src/components/MarkdownProvider/components/Usage.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledUsage = styled.div`
+  @media (min-width: ${p => p.theme.breakpoints.sm}) {
+    display: flex;
+  }
+`;
+
+export const Usage: React.FC = props => <StyledUsage {...props} />;
```

Comment:
Is it possible to export `StyledUsage` directly?

Reply:
Could you clarify what is meant by this? I initially assumed you meant that I can export `StyledUsage` directly like this:

```jsx
export const Usage = styled.div`
  @media (min-width: ${p => p.theme.breakpoints.sm}) {
    display: flex;
  }
`;
```

Is this close to what you had in mind?

Reply:
Yep! I would suggest that we keep the `Styled*` naming to help know if we're able to reference them within other styled components.

We can rename it to `Usage` within the `MarkdownProvider` components prop.